lib/ocpp/v201/charge_point.cpp

```diff
@@ -986,8 +979,9 @@ void ChargePoint::update_aligned_data_interval() {
                 }
             }
 
+            // send evseID = 0 values
             const auto meter_value =
-                get_latest_meter_value_filtered(this->get_meter_value(), ReadingContextEnum::Sample_Clock,
+                get_latest_meter_value_filtered(this->aligned_data_evse0.get_values(), ReadingContextEnum::Sample_Clock,
```

Comment:
Are you clearing the evse0 averaging as well?

Maybe you can still do the clearing in the get so you don't always have to group a `get` and `clear` function together. Only `get` is probably not the best name then since you will be changing the internal state which you normally don't do on a get